src/libraries/Common/tests/TestUtilities/System/PlatformDetection.cs

```diff
@@ -61,6 +61,9 @@ public static partial class PlatformDetection
         public static bool IsUsingLimitedCultures => !IsNotMobile;
         public static bool IsNotUsingLimitedCultures => IsNotMobile;
 
+        public static bool IsPlatformCryptoSupported => !IsBrowser || IsBrowserDomSupported;
+        public static bool IsNotPlatformCryptoSupported => !IsPlatformCryptoSupported;
```

Comment:
Is this ever true for real experiences, or just our current test harness/emulator?  (If it's true in real experiences, then having a test that shows we have some sort of sensible failure path seems virtuous)